jasmine/build/bundle.js

```diff
@@ -52,15 +52,15 @@ describe('Inverted Index', () => {
       expect(Object.prototype.hasOwnProperty.call(myMap, 'books.json'))
```

Comment:
The `bundle.js` file is generate using a task runner, therefore shouldn't be committed to the repo.

Executing your tasks on deployment would also ensure the file is generated on the host server.

Take note.